src/types.ts

```diff
@@ -287,6 +287,7 @@ export interface QueryBuilderProps {
    */
   combinators?: NameLabelPair[];
   controlElements?: Partial<Controls>;
+  enableMountQueryChange: boolean;
```

Comment:
I believe you need a `?` here so that it's optional.

Reply:
Correct. Updated. 👍